matter_server/server/__main__.py

```diff
@@ -103,6 +104,12 @@
     action="store_true",
     help="Enable PAA root certificates and other device information from test-net DCL.",
 )
+parser.add_argument(
+    "--log-node-ids",
+    type=int,
+    nargs="+",
+    help="List of node IDs to show logs from (applies only to server logs).",
+)
```

Comment:
Can we somehow turn this into a runtime config ?

Reply:
I was thinking of exposing it as ws command and make a nice button in the debug UI

Reply:
The downside of that is logging during startup then is unfiltered. I kinda like the add-on config approach since it sets up the filter at the very start.

The feature also might be used by relatively non-technical people. I'd prefer if we don't have to lead them to the debug UI just to get a startup log for a particular problematic node. 

How about adding the runtime log filter in a follow up?